src/styles/blocks/page.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  &__body {
+    margin: 0;
+  }
+
+  &__stopwatch {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%,-50%);
```

Comment:
as i said before, no need to use both top/left and translate

Reply:
Top, left - takes parent sizes, transform: translate - takes block sizes. Works fine when body height = 0px .
Flexbox method "align-items: center" only works when I set body "height: 100vh", otherwise it fails.
![flex-align-justify](https://github.com/mate-academy/layout_stop-watch/assets/91250208/7c392a1f-7a03-4a36-b3bf-82f256e8cd77)